samples/bot-calling-meeting/csharp/Source/CallingBotSample/Bots/MessageBot.cs

```diff
@@ -122,125 +124,89 @@ protected override async Task<TaskModuleResponse> OnTeamsTaskModuleSubmitAsync(I
             var moduleSubmitData = asJobject.ToObject<TaskModuleSubmitData>();
             var peoplePicker = moduleSubmitData?.PeoplePicker;
 
-            var conversationStateAccessors = conversationState.CreateProperty<MeetingActionDetails>(nameof(MeetingActionDetails));
-            var conversationData = await conversationStateAccessors.GetAsync(turnContext, () => new MeetingActionDetails());
-
             if (peoplePicker != null)
             {
+                // Adaptive Card people picker returns a comma separated list of aad IDs
```

Comment:
Thanks for the comment-- this was a helpful reminder of what comes back from a people picker.